src/main/java/com/app/onlinebookstore/model/Category.java

```diff
@@ -0,0 +1,34 @@
+package com.app.onlinebookstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Data;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.Where;
+
+@Entity
+@SQLDelete(sql = "UPDATE categories SET is_delete = true WHERE id =?")
+@Where(clause = "is_delete=false")
+@Data
+@Table(name = "categories")
+public class Category {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Column(nullable = false)
+    private String name;
+    private String description;
+    @Column(nullable = false)
+    private boolean isDelete = false;
+
+    public Category() {

```

Comment:
Do you really need constructors?

Reply:
Yes, becouse i have constructor with parametrs